cadence/transactions/mint_nft.cdc

```diff
@@ -1,12 +1,12 @@
-import NonFungibleToken from "../../contracts/NonFungibleToken.cdc"
-import GametvNFT from "../../contracts/GametvNFT.cdc"
+import NonFungibleToken from "../contracts/NonFungibleToken.cdc"
+import GametvNFT from "../contracts/GametvNFT.cdc"
 
 // This transction uses the NFTMinter resource to mint a new NFT.
 //
 // It must be run with the account that has the minter resource
 // stored at path /storage/NFTMinter.
 
-transaction(recipient: Address, typeID: String) {
+transaction(metadata: {String: AnyStruct}, recipient: Address, typeID: String) {
```

Comment:
Change to `transaction(recipient: Address, typeID: String, metadata: {String: AnyStruct}) {`
Note the order of params